index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div class="container">
+      <div class="row">
+          <div class="col-md-2">
+            <nav id="navbar stick" >
+              <header>JS Documentationn</header>
+              <a href="#Introduction" class="nav-link">Introduction</a>
+              <a href="#What_You_Know" class="nav-link">What you should already know</a>
+              <a href="#Javascript_&_Java" class="nav-link">JavaScript and Java</a>
+              <a href="#Hello_World" class="nav-link">Hello world</a>
+              <a href="#Variables" class="nav-link">Variables</a>
+            </nav>
+          </div>
+          <div class="col-md-10">
+            <main id="main-doc">
+            <section class="main-section" id="Introduction">
+               <header>
+                <h1>Introduction</h1>
+             </header>
```

Comment:
I think that you should add a comment which is clear enough and add the reason why the student needs to add the indentation as it may help them understand if they had no idea about it before. 

This review is polite but could have been more polite and a little less like an order if you follow the You can read more about professional communication in this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate).